rabbus.go

```diff
@@ -42,16 +44,29 @@ type Config struct {
 	Dsn string
 	// Durable indicates of the queue will survive broker restarts. Default to true.
 	Durable bool
+	// PassiveExchange forces passive connection with all exchanges using
+	// amqp's ExchangeDeclarePassive instead the default ExchangeDeclare
+	PassiveExchange bool
+	// Qos controls how many messages or how many bytes will be consumed before receiving delivery acks
+	Qos
+	// Retry settings for the in memory retry mechanism
+	Retry
+	// Breaker circuit breaker configuration
+	Breaker
+}
+
+// Retry config for the in memory retry mechanism
+type Retry struct {
 	// Attempts is the max number of retries on broker outages.
 	Attempts int
 	// Sleep is the sleep time of the retry mechanism.
 	Sleep time.Duration
+}
+
```

Comment:
Could you add a description about this struct?

Reply:
Thx!